Libraries/Utilities/deepFreezeAndThrowOnMutationInDev.js

```diff
@@ -38,7 +38,10 @@ function deepFreezeAndThrowOnMutationInDev(object: Object) {
       return;
     }
 
-    for (var key in object) {
+    var keys = Object.keys(object);
+
+    for (var i = 0; i < keys.length; i++) {
```

Comment:
Because you copied the keys from the object, i.e.: there is no direct reference anymore, I think you can still use the enhanced for loop. So it could look like this: 
`var keys = Object.keys(object);
for (var key in keys) { 
 if (object.hasOwnProperty(key)) {
          object.__defineGetter__(key, identity.bind(null, object[key]));
          ...
}}`